pagopa/io-app
# [#174179069] Pin insertion overflow

**Short description:**
Fix the app pin insertion overflow

**How to reproduce the bug:**

- insert the pin (1st time) `333333`
- insert the same pin of the 1st time but adding 2 digits more `33333344`
- the _continue_ button is displayed (ok, that's right)
- press del _delete_ button: _continue_ button disappears but the pin seems full filled (bug)

<img src="https://user-images.githubusercontent.com/822471/89338513-fa409480-d69c-11ea-997f-aa3dd7c53cfa.png" height="600"/>
